01week/rockPaperScissors.js

```diff
@@ -9,8 +9,17 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
+  if (hand1 === hand2) {
```

Comment:
Looking really good. Since you have this down so well, find a way to get rid of the / in the console when the outcome is a tie then see if you can decompose into separate functions to make your code more stable.